sentry_auth_github/client.py

```diff
@@ -35,7 +35,7 @@ def _request(self, path, access_token):
                 headers=headers,
             )
         except RequestException as e:
-            raise GitHubApiError(unicode(e), status=e.status_code)
+            raise GitHubApiError(unicode(e), status=getattr(e, 'status_code', None))
```

Comment:
if you look at the `__init__` method of the `GitHubApiError`, status defaults to None: https://github.com/getsentry/sentry-auth-github/pull/13/files/1c1a0184f212d23eb160f1a48be755cb097491f1#diff-61801059f36d3f762c56c216c6940d9eL11

so, if i change it here, should i change it there too?